app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
+    url = f"http://api.weatherapi.com/v1/current.json?key={api_key}"
+    filtering = "Paris"
+
+    print(f"Performing request to Weather API for city {filtering}...")
+    response = requests.get(url + f"&q={filtering}")
+    data = response.json()
```

Comment:
It's a good practice to check if the response is successful before calling `response.json()`. You can use `response.raise_for_status()` to raise an error for unsuccessful requests.